### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by swarmer**

### CODE REVIEW

File: 2015/swarmer/task1/Gemfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'curses'
```
Comment (Saicheg) [inline 2015/swarmer/task1/Gemfile]: :+1: for Gemfile!

File: 2015/swarmer/task1/sl.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  def render(window)
+    lines.each.with_index do |line, i|
+      line.each_char.with_index do |char, j|
+        next if char == ' '
+
+        cy = i + y
+        cx = j + x
+        if !(0...window.maxy).include?(cy) || !(0...window.maxx).include?(cx)
```
Comment (Saicheg) [inline 2015/swarmer/task1/sl.rb]: this can be extracted to separate method

**Comment by Saicheg** [comment]
in overral - looks great :+1:

**Comment by swarmer** [comment]
Extracted bounds checking into a separate method.

**Pull request merged**